process/block/baseProcess.go

```diff
@@ -1270,13 +1274,11 @@ func (bp *baseProcessor) saveBody(body *block.Body, header data.HeaderHandler, h
 		log.Trace("saveBody.Put -> MiniBlockUnit", "time", time.Since(startTime), "hash", miniBlockHash)
 	}
 
-	process.SaveReceipts(process.ArgsSaveReceipts{
-		MarshalizedReceiptsProvider: bp.txCoordinator,
-		Marshaller:                  bp.marshalizer,
-		Hasher:                      bp.hasher,
-		Store:                       bp.store,
-		Header:                      header,
-	})
+	receiptsHolder := &process.ReceiptsHolder{Miniblocks: bp.txCoordinator.GetCreatedInShardMiniBlocks()}
+	errNotCritical = bp.receiptsRepository.SaveReceipts(receiptsHolder, header, headerHash)
+	if errNotCritical != nil {
+		log.Warn("saveBody(), error on receiptsRepository.SaveReceipts()", "error", errNotCritical.Error())
```

Comment:
please treat the log level accordingly based on the error type.
```GO
level = logger.LogWarn
if errors.IsClosingError(errNotCritical) {
   level = logger.LogDebug
}

log.Log(level, "saveBody(), error on receiptsRepository.SaveReceipts()", "error", errNotCritical.Error())
```

Reply:
Fixed to use `handleErrorLogging` (for some other errors in this file, as well).

Reply:
If the new utility function is all right, I can change existing code (in other places) to use it.

Reply:
Looks good :+1: